submission/programs/adae.R

```diff
@@ -0,0 +1,350 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+# Output: adae.xpt
+#
+# Developed by : [Author/Developer name]
+
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(curl)
```

Comment:
Are you using curl? If not please remove.

Reply:
Removed. Please review and resolve to ensure you do not see it on your end.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.